components/Elevation/tests/unit/MDCMaterialElevationRespondingTests.m

```diff
@@ -457,4 +461,49 @@ - (void)testElevationOverrideViewControllerInUIView {
                              fakeElevationOverride, 0.001);
 }
 
+#pragma mark - Elevation did change
+
+- (void)testElevationDidChangeCallsElevationDidChangeBlockAndCorrectParameters {
+  // Given
+  CGFloat fakeElevation = 3;
+  self.elevationOverrideView.mdc_overrideBaseElevation = fakeElevation;
+  [self.elevationOverrideView addSubview:self.elevationView];
+  __block CGFloat passedElevation = -1;
+  __block id<MDCElevatable> passedObject = nil;
+  XCTestExpectation *expectation =
+      [[XCTestExpectation alloc] initWithDescription:@"elevationDidChange"];
+  self.elevationView.mdc_elevationDidChangeBlock =
+      ^(id<MDCElevatable> _Nonnull elevatableSelf, CGFloat elevation) {
+        passedElevation = elevation;
+        passedObject = elevatableSelf;
+        [expectation fulfill];
+      };
+
+  // When
+  [self.elevationView mdc_elevationDidChange];
+
+  // Then
+  [self waitForExpectations:@[ expectation ] timeout:1];
+  XCTAssertEqualWithAccuracy(passedElevation, fakeElevation, 0.001);
+  XCTAssertEqual(passedObject, self.elevationView);
+}
+
+- (void)testElevationDidChangeCallsSubviews {
```

Comment:
Please add a test that the value of `mdc_baseElevation` on the superview is updated by the time the subview receives the call to `elevationDidChange:`.  That will help verify that the implementation is calling the blocks on the superviews before calling the blocks on the subviews.

Something like this:

```objc
XCTestExpectation *expectation = [self expectationWithDescription:@("Override view elevation block was called.")];
__block CGFloat viewBaseElevation = -1;
[self.view addSubview:self.elevationOverrideView];
[self.elevationOverrideView addSubview:self.elevationView];
self.elevationView.mdc_elevationDidChangeBlock = ^(id<MDCElevatable> object, CGFloat elevation) {
  viewBaseElevation = object.mdc_baseElevation;
};
self.elevationOverrideView.mdc_elevationDidChangeBlock = ^(id<MDCElevatable> object, CGFloat elevation) {
  if ([object conformsToProtocol:@protocol(MDCElevationOverriding)]) {
    id<MDCElevationOverriding> objectAsOverriding = (id<MDCElevationOverriding>)object;
    objectAsOverriding.mdc_overrideBaseElevation = 99;
    [expectation fulfill];
  }
}

// When
[self.view mdc_elevationDidChange];

// Then
XCTAssertEqualWithAccuracy(viewBaseElevation, 99, 0.001);
```

Reply:
Added the test.